### ISSUE TITLE
Illegal characters in path

### ISSUE THREAD

**Issue by jonmpqts** [mdemr-systems/thicket#1]
When a patient's first or last name has characters in it that are not valid in a file/path name, a file will fail to be produced. @schemie should we just remove these when writing them out? I don't see that having these characters in the filename matters all that much.

**Comment by schemie** [comment]
Yes please scrub anything illegal in the file path out. It will still be intact inside the CCDA itself so it shouldn't be an issue.
